examples/allocation/tinygo/testdata/greet.go

```diff
@@ -53,7 +55,7 @@ func _greet(ptr, size uint32) {
 func _greeting(ptr, size uint32) (ptrSize uint64) {
```

Comment:
correct me if I'm wrong. It seems like in your bench of CGO vs tiny, both call into the same function? Did you mean to have a different guest wasm. e.g. one with tinymem approach which uses an alive pointers map and one that doesn't by using malloc free directly?

Reply:
This is the "old" example, which I reverted to be the same as before, then I added a new example: `examples/allocation/tinygo-malloc` which exports `_greeting_cgo` and `_greeting_tinymem` 

Note: names must probably be changed to something more meaningful

Reply:
please revert this name, but keep the comment. It is still wasm memory, so it isn't native. I think the change is actually more confusing than before.

Reply:
the comment ended up on the wrong line somehow. reposting it.

Reply:
done